src/current/_includes/v22.2/orchestration/kubernetes-stop-cluster.md

```diff
@@ -128,7 +129,7 @@ This does not delete any secrets you may have created. For more information on m
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    $ kubectl delete csr default.client.root default.node.my-release-cockroachdb-0 default.node.my-release-cockroachdb-1 default.node.my-release-cockroachdb-2 default.node.my-release-cockroachdb-3
+    kubectl delete csr default.client.root default.node.my-release-cockroachdb-0 default.node.my-release-cockroachdb-1 default.node.my-release-cockroachdb-2 default.node.my-release-cockroachdb-3
```

Comment:
This will no longer be relevant if cert-manager is used to create certificates. Perhaps replace this with deleting the Issuer created in cert-manager. For example:

kubectl get issuer

kubectl delete issuer cockroachdb

Reply:
You're right, and we only updated the Helm docs to use `cert-manager`. This will take a bit more time to update the Operator and manual StatefulSet docs to use `cert-manager`.